source/declarations.tex

```diff
@@ -5868,8 +5868,8 @@
 int x = 999;                    // \tcode{x} is not a constant expression
 const int y = 999;
 const int z = 99;
-char c1 = x;                    // OK, though it might narrow (in this case, it does narrow)
-char c2{x};                     // error: might narrow
+char c1 = x;                    // OK, though it can narrow (in this case, it does narrow)
+char c2{x};                     // error: can narrow
```

Comment:
I don't think "can" makes sense here, because the code is ill-formed, so it can't do anything. How about "error: potentially narrows"?

Reply:
fixed